Helpers.gs

```diff
@@ -355,10 +355,14 @@ function createEvent(event, calendarTz){
   if (event.hasProperty('location'))
     newEvent.location = icalEvent.location;
 
-  if (event.hasProperty('class')){
-    var classString = event.getFirstPropertyValue('class').toString().toLowerCase();
-    if (["default", "public", "private", "confidential"].indexOf(classString) > -1)
-      newEvent.visibility = classString;
+  if ( ["default", "public", "private"].includes(forceVisibility.toLowerCase()) ) {
```

Comment:
Why does this array not include `confidential` like the array on 363? I think better to add that value and break out the array to something like `var validVisibilityValues` and use it in both places

Reply:
Agreed. Changes committed.